dataprofiler/tests/profilers/profiler_options/test_profiler_options.py

```diff
@@ -101,7 +101,8 @@ def test_numerical_stats_option(self, *mocks):
                     profile_column["statistics"]["histogram"]["bin_edges"])
                 self.assertIsNotNone(profile_column["statistics"]["min"])
                 self.assertIsNotNone(profile_column["statistics"]["max"])
-                self.assertNotEqual(0, profile_column["statistics"]["variance"])
+                print(profile_column["statistics"]["variance"])
+                self.assertEqual(0.5, profile_column["statistics"]["variance"])
```

Comment:
You could also just do `self.assertFalse(np.isnan(profile_column["statistics"]["variance"]))` if all you care about is whether or not it is nan, although its not a big deal.

Reply:
will approve as is, but i think the original intent is we just care in options that it isn't the default value anymore, but this is fine too.